Misc/reputations.json

```diff
@@ -121,6 +121,15 @@
       "reputationScriptName": "cveReputation",
       "details": "CVE CVSS Score",
       "enhancementScriptNames": ["cveSearch"]
+    },
+    {
+      "id": "userRep",
+      "version": -1,
+      "regex": "",
+      "reputationScriptName": "",
+      "details": "user",
+      "enhancementScriptNames": []
     }
-  ]
+  ],
+  "releaseNotes":"add new user reputation"
```

Comment:
Sentence should start with a capital letter.

Reply:
@meirwah 
1. Make release notes more descriptive
2. we should add a command/script to calculate the reputation, if not at least enhancement scripts